pyproject.toml

```diff
@@ -17,11 +17,18 @@ readme = "README.md"
 dynamic = ["version"]
 
 dependencies = [
-    "pydantic == 2.6.0",
-    "semver == 3.0.2"
+    "pydantic == 2.7.0",
+    "semver == 3.0.2",
+    "jinja2 == 3.1.3",
+    "boto3 == 1.34.95"
 ]
 
 [project.optional-dependencies]
+
+vr_foraging = [
+    "aind_behavior_vr_foraging@git+https://github.com/AllenNeuralDynamics/Aind.Behavior.VrForaging@main",
```

Comment:
As I mentioned before, I think this should be removed from the main branch. I make not commitments to maintain the main branch of vr-foraging to be forever compatible with this curriculum example :P

Reply:
in that case, removed the vr-foraging example as well. The 2 examples provided should be sufficient